Documentation/install/upgrade.rst

```diff
@@ -618,6 +623,77 @@ IMPORTANT: Changes required before upgrading to 1.7.0
 New ConfigMap Options
 ~~~~~~~~~~~~~~~~~~~~~
 
+  * ``enable-remote-node-identity`` has been added to enable a new identity
+    for remote cluster nodes. This allows for network policies that distinguish
+    between connections from host networking pods or other processes on the local
+    Kubernetes worker node from those on remote worker nodes.  This is important
+    because Kubernetes Network Policy dictates that network connectivity from the
+    local host must always be allowed, even for pods that have a default deny rule for
+    ingress connectivity.   This is so that network liveness and readiness
+    probes from kubelet will not be dropped by network policy.  Prior to 1.7.x,
+    Cilium achieved this by always allowing ingress host network connectivity from any
+    host in the cluster.  With 1.7 and ``enable-remote-node-identity=true``, Cilium
+    will only automatically allow connectivity from the local node, thereby providing
+    a better default security posture.
+
+    The option is enabled by
+    default for new deployments when generated via Helm, in order to gain the benefits
+    of improved security. The option can be disabled
+    in order to maintain full compatibility with Cilium 1.6.x policy enforcement.
+    **Be aware** that upgrading a cluster to 1.7.x by using helm to generating a new Cilium config
+    that leaves ``enable-remote-node-identity`` set as the default value
+    of ``true`` **can break network connectivity.**
+
+    The reason for this is that
+    with Cilium 1.6.x, the source identity of ANY connection from a host-networking pod or from
+    other processes on a Kubernetes worker node would be the  ``host`` identity.   Thus, a
+    Cilium 1.6.x or earlier environment with network policy enforced may be implicitly
+    relying on the ``allow everything from host identity`` behavior to
+    whitelist traffic from host networking to other Cilium-managed pods.
+    With the shift to 1.7.x, if ``enable-remote-node-identity=true``
+    these connections will be denied by policy if they are coming from
+    a host-networking pod or process on another Kubernetes worker node, since the source
+    will be given the ``remote-node`` identity (which is not automatically
+    allowed) rather than the ``host`` identity (which is automatically allowed).
+
+
```

Comment:
Most of the file uses single lines as paragraph delimiters, maybe avoid double blank lines?